migrations/20210420154714-create-shopper.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       UserId: {
-        type: Sequelize.STRING
+        type: Sequelize.INTEGER
```

Comment:
This type was incorrect for the UserId. I fixed it. For things to work, you MUST run

```
sequelize db:migrate:undo
```

Run that command as many time as you need until in undoes the create-shopper migration. Then, re-run the migrations forward:

```
sequelize db:migrate
```